From c2225e463aace57a5e1de9b70d5410c1620bc3ce Mon Sep 17 00:00:00 2001
From: James Slagle <jslagle@redhat.com>
Date: Thu, 13 Feb 2014 17:06:34 -0500
Subject: [PATCH 08/12] Install lvm2 package for cinder

Adds lvm2 package to get installed for the cinder package install, it's
required for our migration.d script.

Change-Id: Ie18a9bf3e82983888cad33aef9dc0ab56afe1374
---
 elements/cinder/install.d/cinder-package-install/72-cinder | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/elements/cinder/install.d/cinder-package-install/72-cinder b/elements/cinder/install.d/cinder-package-install/72-cinder
index 7c7dfe3..5f85b37 100755
--- a/elements/cinder/install.d/cinder-package-install/72-cinder
+++ b/elements/cinder/install.d/cinder-package-install/72-cinder
@@ -2,4 +2,6 @@
 
 set -eux
 
-install-packages openstack-cinder
+# lvm2 is required by our cinder script in migration.d, but not by the cinder
+# package itself.
+install-packages openstack-cinder lvm2
-- 
1.8.5.3

